docker-compose.yml

```diff
@@ -57,3 +59,18 @@ pg:
     - "5432"
   volumes:
     - ./test/db/:/docker-entrypoint-initdb.d/
+elasticsearch:
+  image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.3.0
+  container_name: elasticsearch
+  environment:
+    - cluster.name=docker-cluster
+    - bootstrap.memory_lock=true
+    - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
```

Comment:
Hmm... maybe these other ENVs are no longer necessary, if this changes to single-node mode.  (And should change this one to match the docker-compose-ci)

Reply:
yeah I think I cargo-culted most of the env stuff from an ES example.